mwclient/client.py

```diff
@@ -50,6 +50,9 @@ class Site(object):
         >>> site = mwclient.Site('sourceforge.net', path='/apps/mediawiki/mwclient/')
 
     """
+    AVAILABLE_TOKENS_TYPES = {
```

Comment:
TOKEN_TYPES, not TOKENS_TYPES perhaps ? And while we're touching this, perhaps the list should also be sorted alphabetically for better readability?

Reply:
done